typings/src/globals/globals.ts

```diff
@@ -0,0 +1,98 @@
+import { hrTestData, IHRTestData } from './test_data/test_data';
+
+interface IApp {
+    financials: {
+        adminUserId: string;
+        loginPassword: string;
+    };
+}
+
+const appConfig: IApp = {
+    financials: {
+        /** Financials admin user id for logging into financials */
+        adminUserId: `${process.env.FINANCIALS_ADMIN_USER_ID}`,
+        /** Password used for logging into financials. */
+        loginPassword: `${process.env.FINANCIALS_TEST_PASSWORD}`
+    }
```

Comment:
rename `financials` to `coinflex` and `FINANCIALS_ADMIN_USER_ID` to `USER_ID`. Same for `FINANCIALS_TEST_PASSWORD` -> `PASSWORD`